tests/csm/rest/test_capacity_quota.py

```diff
@@ -641,3 +646,354 @@ def test_40640(self):
         res = self.csm_obj.verify_max_objects(max_size, max_objects, self.akey, self.skey)
         assert res[0], res[1]
         self.log.info("##### Test ended -  %s #####", test_case_name)
+
+    @pytest.mark.skip("Feature not ready")
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-41127')
+    def test_41127(self):
+        """
+        Test GET capacity stats for create IAM user and Put objects using admin users
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        test_cfg = self.csm_conf["test_41127"]
+        max_size = test_cfg["max_size"]
+        random_size = self.cryptogen.randrange(1, max_size)
```

Comment:
self.csm_obj.random_gen.randrange() instead of cryptogen

Reply:
changed